### PR TITLE
#60577433 Update TravisCI to run tests

### PR DESCRIPTION
**Pull request by gloriaodipo**
**What does this PR do?**
- It implements continuous integration by integrating TravisCI.

**Description of task to be completed**
- Update TravisCI to run tests.

**How should this be manually tested?**
- Follow this [travisCI link](https://travis-ci.org/andela/ah-shakas-frontend) and choose any branch then manually trigger a build.

**Screenshots**
- The following screenshots show the output to be expected after a build succeeds:

image

image

**What are the relevant Pivotal Tracker stories?**
[#160577433](https://www.pivotaltracker.com/n/projects/2198533/stories/160577433)

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  
+cache:
+  directories:
+    - node_modules
+    
+install:
+  - yarn install
+
+script:
+  - yarn test -- --coverage
```
Comment (lukorito) [inline .travis.yml]: Hey, based on @Arthur236's feedback. Change this to `yarn coverage`

**Review by lukorito — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 1 above)
Comment (gloriaodipo) [reply .travis.yml]: Okay, that's noted

**Comment by lukorito** [comment]
@gloriaodipo update the screenshot

**Review by Arthur236 — APPROVED** [review]
Looks good 👍

**Review by lukorito — APPROVED** [review]

**Review by codingedward — APPROVED** [review]
Looks good to me too 👍.

**Pull request merged**